src/shogun/neuralnets/ConvolutionalFeatureMap.cpp

```diff
@@ -283,6 +286,10 @@ void CConvolutionalFeatureMap::convolve(
 			outputs.matrix+i*outputs.num_rows + outputs_row_offset,
 			m_output_height, m_output_width, false);
 
+#if defined(HAVE_LINALG_LIB)
+		//linalg::convolve<linalg::Backend::VIENNACL>(image, weights, result, flip, reset_output, m_stride_x, m_stride_y, m_autoencoder_position);
+		linalg::convolve(image, weights, result, flip, reset_output, m_stride_x, m_stride_y, m_autoencoder_position);
+#else
 		for (int32_t x=0; x<m_input_width; x+=m_stride_x)
```

Comment:
@karlnapf We have had this discussion before in Issue #2667 (in the end). Don't think C++11 was made a compulsory dependency for shogun.